Apps/Examples/Examples/SwiftUI Examples/SimpleMapExample.swift

```diff
@@ -6,7 +6,7 @@ struct SimpleMapExample: View {
     @State var camera = CameraState(center: .helsinki, zoom: 12)
     var body: some View {
         Map(camera: $camera)
-            .styleURI(.streets, darkMode: .dark)
+            .styleURI(.streets, dark: .dark)
```

Comment:
darkMode seem to be more explicit, it means "style to use for dark mode". if it's just "dark" it seems to be quite general